code/game/objects/items/storage/backpack.dm

```diff
@@ -39,6 +39,7 @@
 	STR.max_volume = STORAGE_VOLUME_BACKPACK
 	STR.max_w_class = MAX_WEIGHT_CLASS_BACKPACK
 	STR.use_sound = 'sound/items/storage/unzip.ogg'
+	STR.worn_access = FALSE
```

Comment:
satchels are a subtype of backpacks & need stuff updated

Reply:
👍